src/clj_kondo/impl/linters/deps_edn.clj

```diff
@@ -188,8 +188,10 @@
   (let [tasks (edn-utils/sexpr-keys expr)
         known-task? (set (keys tasks))]
     (doseq [[_ t-def] tasks
-            dep-task   (:children (last (:children t-def)))
+            [td-key td-body]   (partition 2 (:children t-def))
```

Comment:
you could also use `sexpr/keys` here and just get the `:depends` key out of that perhaps.